library/src/main/java/com/anjlab/android/iab/v3/BillingProcessor.java

```diff
@@ -296,10 +297,14 @@ private boolean loadPurchasesByType(String type, BillingCache cacheStorage)
 		return false;
 	}
 
+	/**
+	 * Attempt to fetch purchases from the server and update our cache if successful
+	 * @return {@code true} if retrieval occurs, {@code false} otherwise
+	 * todo check if loading should return true if response code isn't successful
```

Comment:
it does returns false when unsuccessful from here: https://github.com/anjlab/android-inapp-billing-v3/blob/master/library/src/main/java/com/anjlab/android/iab/v3/BillingProcessor.java#L296